pkg/kubernetes/app.js

```diff
@@ -72,6 +94,22 @@ define([
                 };
         }])
 
+        /* Call selectpicker after last value rendered */
+        .directive('selectWatcher', ["$timeout",
```

Comment:
I think this will need to wait for #2569. angular-patternfly has versions that support 1.1.x and 2.x but not the version of patternfly we are using now 1.3